test/system/react_components/notifications/icon_test.rb

```diff
@@ -0,0 +1,23 @@
+require "application_system_test_case"
+
+module Notifications
```

Comment:
Maybe we should namespace this under `ReactComponents`. I just copied how we have it right now.

Reply:
Yeah, namespacing sounds good.

In #55 I've renamed this folder to be `Components` FYI.

Reply:
Components I like better too, especially if you're going Vue or whatever at some time.

Reply:
Let's do this in a follow-up PR!